pcsx2/IPU/IPU.cpp

```diff
@@ -243,6 +243,10 @@ __fi u64 ipuRead64(u32 mem)
 		ipucase(IPU_CMD): // IPU_CMD
 			if (ipuRegs.cmd.DATA & 0xffffff)
 				IPU_LOG("read64: IPU_CMD=BUSY=%x, DATA=%08X", ipuRegs.cmd.BUSY ? 1 : 0, ipuRegs.cmd.DATA);
+			if (ipuRegs.cmd.BUSY && ipuRegs.cmd.DATA == 0x000001B7) { //prevents Enthusia - Professional Racing end of video crash.
```

Comment:
> I'm not sure if it's a command or a result thereof, but 0x000001B7 is always the final data of a video, in all the games I've tested. And in all those games, ipuRegs.cmd.BUSY is no longer busy, as the video ends and the IPU gets reset. This does not automatically happen in Enthusia, so it needs to do done manually.

This sounds like an almost perfect comment to me. Just also list the games which you tried, and it's great.